src/modules/core/bump.listener.ts

```diff
@@ -144,21 +145,23 @@ export async function handleBump(
 }
 
 export const BumpListener: EventListener = {
-	ready: async (client) => {
+	clientReady: async (client) => {
 		scheduleBumpReminder(client);
 	},
 	messageCreate: async (client, message) => {
 		const interaction = message.interactionMetadata;
 
+		if (!interaction || interaction.type !== InteractionType.ApplicationCommand)
+			return;
+		if (message.author.id !== "302050872383242240") return; // /disboard user id
+		const interactionOld = message.interactionMetadata;
+
+		// instead of checking the command name, we check the description
 		if (
-			!interaction ||
-			!(interaction.type === InteractionType.ApplicationCommand)
+			interactionOld?.type !== InteractionType.ApplicationCommand ||
+			!message.embeds[0]?.description?.includes("Bump done")
```

Comment:
have you verified that this works? the embed checking is _very_ fragile and difficult to test, so I would recommend against changing its code unless you're 100% certain it works and is an improvement in some way